### PR TITLE
Move Replicant's Redmine to a docker container

### PR DESCRIPTION
**Pull request by EMH333**
Still need to add the Replicant DB credentials to the databags repo and move the files over to the new location but the rest of it should be good to go here

### CODE REVIEW

**Comment by EMH333** [comment]
[#31250: Updating Replicant's Redmine](https://trello.com/c/mMnq0trz/1876-31250-updating-replicants-redmine)

File: resources/app.rb [hunk 1]
```diff
@@ -48,3 +48,9 @@
     action [:stop, :delete]
   end
 end
+
+action :stop do
+  systemd_service new_resource.service_name do
+    action [:stop, :disable]
+  end
+end
```
Comment (ramereth) [inline resources/app.rb:56]: This should be split up into two distinct actions, ``:stop`` and ``:disable``.

File: recipes/app2.rb [hunk 2]
```diff
@@ -98,4 +102,40 @@
   environment 'RAILS_ENV' => 'production'
   working_directory '/home/replicant/redmine'
   pid_file '/home/replicant/redmine/pids/unicorn.pid'
+  action :stop
```
Comment (ramereth) [inline recipes/app2.rb:105]: Once you apply the fix I suggested below, you should update this to the following. You might also add a ``# TODO:`` comment here to remove this after the migration has been completed.
```suggestion
  action [:stop, :disable]
```

File: spec/app2_spec.rb [hunk 3]
```diff
@@ -154,10 +154,26 @@
     )
   end
 
+  it do
+    expect(chef_run).to disable_osl_app('replicant-redmine-unicorn').with(
+      user: 'replicant',
```
Comment (ramereth) [inline spec/app2_spec.rb:159]: For this one, don't worry about all the parameters since you already tested that above.

**Comment by EMH333** [comment]
!bump minor *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v2.6.0.  Have a nice day!

**Pull request merged**